src/helpers/xml-builder.js

```diff
@@ -9,9 +9,8 @@ import isVNode from 'virtual-dom/vnode/is-vnode';
 import isVText from 'virtual-dom/vnode/is-vtext';
 import colorNames from 'color-name';
 import { cloneDeep } from 'lodash';
-import imageToBase64 from 'image-to-base64';
 import sizeOf from 'image-size';
-import { getMimeType } from '../utils/image'
+import { getMimeType, downloadImageToBase64 } from '../utils/image';
```

Comment:
Would suggest not to make refactor changes in the same PR,  as it does not add value and creates confusion during review.

Small changes work but larger changes like these add complexity.